tmt/steps/provision/__init__.py

```diff
@@ -355,11 +355,17 @@ def details(self):
         self.verbose('kernel', kernel, 'green')
 
     def _ansible_verbosity(self):
-        """ Prepare verbose level based on the --debug option count """
+        """ Prepare verbose level based on the --debug or --verbose option
+         count or verbose attribute """
         if self.opt('debug') < 3:
             return ''
         else:
-            return ' -' + (self.opt('debug') - 2) * 'v'
+            if self.parent.data[0].get('verbose'):
+                return ' -' + max(self.opt('verbose'),
+                                  self.parent.data[0].get('verbose')) * 'v'
```

Comment:
I don't think it is a good idea to directly access parent data in a raw form from here. Rather I'd suggest to provide an extra parameter to the `guest.ansible()` method and call if from the `ansible` plugin.